opentuner/search/technique.py

```diff
@@ -204,15 +223,20 @@ def call_main_generator(self):
                                     self.pending_tests)
         if self.pending_tests:
           self.rounds_since_novel_request = 0
-          yield None # wait
+          yield False # wait
 
 #list of all techniques
 the_registry = list()
 
+#list of composable techniques
+the_composable_registry = list()
 
 def register(t):
   the_registry.append(t)
 
+def register_composable(t):
```

Comment:
I'm not sure I understand what you mean.
Would making an optional argument (i.e. register(t, composable=False) ) be okay? 

My thinking is that a user could register multiple C.E.T. instances of the same class with different hyperparameters (population size, crossover rate, etc) and operators specified. However, when we generate a C.E.T. randomly, we want to randomly choose from the available C.E.T. classes, not the available instances.  The composable registry is representing that set of classes (except it contains default instances)